File: patterns/c-patterns/pattern135.c
// Pattern 135: ...

// * * * * *
// *
// * * * * *
//         *
// * * * * *

#include <stdio.h>

int main()
{
    int n = 5;
    int mid = n / 2 + 1;
    int i, j;

    for (i = 1; i <= n; i++)
    {
        for (j = 1; j <= n; j++)
        {
            if (i == 1 || i == n || i == mid || (j == 1 && i <= mid) || (j == n && i > mid))
            {
                printf("* ");
            }
            else
            {
                printf("  ");
            }
        }
        printf("\n");
    }
    return 0;
}

File: patterns/javascript-patterns/pattern135.js
// Pattern 135: ...

// * * * * *
// *
// * * * * *
//         *
// * * * * *

let n = 5;
let mid = Math.floor(n / 2) + 1;

for (let i = 1; i <= n; i++) {
    let row = '';
    for (let j = 1; j <= n; j++) {
        if (i === 1 || i === n || i === mid || (j === 1 && i <= mid) || (j === n && i > mid)) {
            row += '* ';
        } else {
            row += '  ';
        }
    }
    console.log(row);
}
